app/jobs/make_phone_calls_job.rb

```diff
@@ -0,0 +1,28 @@
+class MakePhoneCallsJob < ApplicationJob
+  queue_as :phone_calls
+
+  require 'net/http'
+  require 'open-uri'
+
+  def perform(args)
+    # Find all the phone stores for which there an activecalendar for the current day of the week
+    # For each store, create a job to make the request
+
+    Rails.logger.debug("Going to make a phone call to #{args[:phone]}")
+
+    phone = args[:phone]
```

Comment:
Since the phone number is mandatory, I would use `args.fetch(:phone)` here, so that the exception is raised before we do the http call